docs/src/community/index.rst

```diff
@@ -46,3 +47,23 @@ smoother interoperability:
    :hidden:
 
    iris_xarray
+
+Plugins
+-------
+
+Iris supports *plugins* under the `iris.plugins` `namespace package`_.  This
+allows packages that extend Iris' functionality to be developed and maintained
+independently, while still being installed into `iris.plugins` instead of a
```

Comment:
```suggestion

Iris supports *plugins* under the ``iris.plugins`` `namespace package`_.  This
allows packages that extend Iris' functionality to be developed and maintained
independently, while still being installed into ``iris.plugins`` instead of a
```

Needs double-ticks to appear as monospace.

---

I'm assuming it would be an anti-pattern to create `iris/plugins/__init__.py`, because that would allow:

```suggestion

Iris supports *plugins* under the :mod:`iris.plugins` `namespace package`_.  This
allows packages that extend Iris' functionality to be developed and maintained
independently, while still being installed into :mod:`iris.plugins` instead of a
```

Reply:
I always forget that rst wants double backticks...

Adding `__init__.py` is not just an anti-pattern, it would mean it's no longer a namespace package.  Perhaps there is a way to have a page for the module in the API documentation anyway?

Reply:
Would probably involve modifying [`generate_package_rst.py`](https://github.com/SciTools/iris/blob/b08cfa63bb435566546d9deb52985ea4561cf391/docs/src/sphinxext/generate_package_rst.py), since we haven't yet migrated to `sphinx-apidoc`. Not worth it IMO - stick with the first option.